wiki/Help_Center/en.md

```diff
@@ -124,15 +118,13 @@ If you believe that you have had your user page access revoked in error, feel fr
 
 ------------------------------------------------------------------------
 
-What are these "user pages" that I see on other players profiles?
------------------------------------------------------------------
+### What are these "user pages" that I see on other players profiles?
 
 osu!supporters are given the privilege of editing their very own page for display on their profile for the world to see. User pages may contain anything you like - so long as it follows the community rules!
 
 ------------------------------------------------------------------------
 
-When I try to register, all the fields say "invalid request"!
--------------------------------------------------------------
+### When I try to register, all the fields say "invalid request"!
 
 This is a known issue, and we aren't yet sure what causes it. Some possible workarounds are:
```

Comment:
Change to
`This has been a known issue; however, we aren't sure what causes this problem. If you see the described issue, try these workarounds:`